algorithms/ar-expsq/java/ExponentiationBySquaring.java

```diff
@@ -0,0 +1,31 @@
+class ExponentiationBySquaring {
+    static long N = 1000000007L; // prime modulo value
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6356801)